resume.md

```diff
@@ -1,49 +1,64 @@
-# FIRST LAST NAME
+# Tejesh Bhaumik
 
-(123) 456 - 7890 | email@gmail.com
+(123) 456 - 7890 | tejeshbhaumik@gmail.com
 
-www.website.com | github.com/myusername | linkedin.com/myusername
+| github.com/Tejesh2001 | linkedin.com/TejeshBhaumik
 
 ## EDUCATION
-**University of College** | B.S. Degree ... *MONTH 20XX*
-* **Coursework**: Data Structures, Algorithms, Upper Division CS Elective, Math Course, Statistics, Data Science, ETC.
+**UIUC** | B.S. of Computer Science + Advertising... *May 2023*
+* **Coursework**: Data Structures, Algorithms, Advanced Linear Algebra, Statistics, Data Science, Probability, Brand Strategy.
 
 ## SKILLS
-* **Languages**: Python, Java, C, SQL, ...
-* **Libraries/Frameworks**: Pandas, TensorFlow, PyTorch, ...
-* **Version Control**: Git, Bitbucket, Sourcetree, GithubDesktop
+* **Languages**: Python, Java, C++, SQL, NodeJs, Android
+* **Libraries/Frameworks**: Pandas, Numpy, Scipy, Sympy
+* **Version Control**: Git, GithubDesktop
 * *...*
 
 ## EXPERIENCE
-**Software Engineer**: Company | City, State ... *Month 20XX - Present*
-* Implemented XYZ Microservice using TECHNOLOGY to delivery PRODUCT to 123 Customers Globally
-* Architected CORE INFRASTRUCTURE LIBRARY used across X teams to BLAH BLAH
-* I did X for Y to accomplish Z
-* ...
+**Android Developer**: Crowd Dynamics Lab   | Remote *September 2020 - Present*
+* Develop the UI of a tracer app using Recycler and Card views in Android studio
```

Comment:
What is a tracer app and why does this matter?  Be more specific.  Good that you're including more technical work.  What do these cards display?  How does this help the end user?